content/pl/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,94 @@
+
+**Era wdrożeń w kontenerach:**
+Kontenery działają w sposób zbliżony do maszyn wirtualnych, ale mają mniejszy stopnień wzajemnej izolacji, współdzieląc ten sam system operacyjny. Kontenery określane są mianem "lekkich". Podobnie, jak maszyna wirtualna, kontener posiada własny system plików, procesor, pamięć, przestrzeń procesów i in. Ponieważ kontenery nie są związane z leżącymi poniżej warstwami infrastruktury, mogą być migrowane pomiędzy chmurami i różnymi dystrybucjami systemu operacyjnego.
+
+Kontenery zyskały popularność ze względu na swoje zalety, takie jak:
+
+* Szybkość i elastyczność w tworzeniu i instalacji aplikacji: obraz kontenera buduje się łatwiej niż obraz VM.
+* *Continuous development, integration, and deployment*: obrazy kontenerów mogą być budowane w sposób wiarygodny i częsty. Wycofanie zmian jst łatwe i szybkie (ponieważ obrazy są niezmienne).
+* Rozdzielenie zadań *Dev* i *Ops*: obrazy kontenerów powstają w fazie *build/release*, oddzielając w ten sposób aplikacje od infrastruktury.
+* Obserwowalność obejmuje nie tylko informacje i metryki z poziomu systemu operacyjnego, ale także poprawność działania samej aplikacji i inne sygnały.
+* Spójność środowiska na etapach rozwoju oprogramowania, testowania i działania w trybie produkcyjnym: działa w ten sam sposób na laptopie i w chmurze.
+* Możliwość przenoszenia pomiędzy systemami operacyjnymi i platformami chmurowymi: Ubuntu, RHEL, CoreOS, on-prem, Google Kubernetes Engine i gdziekolwiek indziej.
+* Zarządzanie, które w centrum uwagi ma aplikacje: Poziom abstrakcji przeniesiony jest z warstwy systemu operacyjnego działającego na maszynie wirtualnej na poziom działania aplikacji, która działa na systemie operacyjnym używając zasobów logicznych.
+* Luźno powiązane, rozproszone i elastyczne "swobodne" mikro serwisy: Aplikacje podzielone są na mniejsze, niezależne komponenty, które mogą być dynamicznie uruchamiane i zarządzane - nie jest to monolityczny system działający na jednej, dużej maszynie dedykowanej na wyłączność.
+* Izolacja zasobów: wydajność aplikacji możliwa do przewidzenia
+* Wykorzystanie zasobów: wysoka wydajność i upakowanie.
+
+## Do czego potrzebujesz Kubernetes i jakie są jego możliwości
+
+Kontenery są dobrą metodą na opakowywanie i uruchamianie aplikacji. W środowisku produkcyjnym musisz zarządzać kontenerami, w których działają aplikacje i pilnować, aby nie było żadnych przerw w ich dostępności. Przykładowo, kiedy jeden z kontenerów przestaje działać, inny musi zostać uruchomiony. Nie byłoby prościej, aby takimi działaniami zajmował się jakiś system?
+
+I tu właśnie Kubernetes przychodzi z pomocą! Kubernetes dostarcza środowisko do pracy w niezawodny sposób z systemami rozproszonymi. Kubernetes obsługuje skalowanie aplikacji, przełączanie w sytuacjach awaryjnych, różne scenariusze wdrożeń itp. Przykładowo, Kubernetes w łatwy sposób może zarządzać wdrożeniem nowej wersji oprogramowania zgodnie z metodyką *canary*.
```

Comment:
For example Helion translates canary as "Wdrażanie kanarkowe"

Reply:
It sounds very weird to me...

Reply:
I'd leave it as is or change to: "zgodnie z metodyką *canary deployment*."

Reply:
fine for me